php7.0/Dockerfile

```diff
@@ -49,6 +51,9 @@ RUN curl \
 RUN mkdir -p /app && rm -fr /var/www/html && ln -s /app /var/www/html
 WORKDIR /app
 
+# Add Composer's vendor/bin/ to the PATH
+ENV PATH "$PATH:/app/vendor/bin"
```

Comment:
I've spent quite some time trying to modify the PATH from `run.sh` but all of my attempts failed. It seems to be only feasible from the Dockerfile, but feel free have a look at my different PATH related commits, and try a different solution.